tools/Nethermind.Tools.Kute/JsonRpcMethodFilter/ComposedJsonRpcMethodFilter.cs

```diff
@@ -15,6 +15,6 @@ public ComposedJsonRpcMethodFilter(IEnumerable<IJsonRpcMethodFilter> filters)
     }
 
     public bool ShouldSubmit(string methodName) =>
-        _filters.IsNullOrEmpty() ||
-        _filters.Any(f => f.ShouldSubmit(methodName));
+        (!_filters?.Any() ?? true) ||
```

Comment:
> Not a fan of !_filters?.Any() ?? true (very hard to understand) but it gets the job done

@emlautarom1 Since I don't maintain this tool, I can apply another approach you like more. As `_filters` is `IEnumerable`, using `Any` is faster than `Count`. Maybe you like more `_filter is null || !_filter.Any()`?

Reply:
Cache to `private readonly` in consructor? HasFilters

Reply:
Alright, refactored and applied @benaadams' suggestion.